src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionTests.cs

```diff
@@ -11145,5 +11145,33 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_BadRetType, "F").WithArguments("Program.F(in System.DateTime)", "string").WithLocation(16, 15)
             );
         }
+
+        [Fact, WorkItem(25813, "https://github.com/dotnet/roslyn/issues/25813")]
+        public void InaccessibleExtensionMethod()
```

Comment:
>InaccessibleExtensionMethod [](start = 20, length = 27)

It would be interesting to test scenario when there are instance candidates and extension candidates and all of them are inaccessible. What would be the error message and candidates returned by SemanticModel?